script.js

```diff
@@ -0,0 +1,462 @@
+                    buttonElement.classList.add('space');
+                    buttonElement.addEventListener('click', () => {
+                        this.properties.value += ' ';
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                    });
+                    break;
+
+                    case "shift":
+                    buttonElement.classList.add('shift');
+                    buttonElement.addEventListener('click', () => {
+                        /*this.properties.value += ' ';*/
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                    });
+                    break;
+
+                    case "rshift":
+                    buttonElement.classList.add('shift');
+                    buttonElement.addEventListener('click', () => {
+                        /*this.properties.value += ' ';*/
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                    });
+                    break;
+
+                    case "ctrl":
+                    buttonElement.classList.add('ctrl');
+                    buttonElement.addEventListener('click', () => {
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                    });
+                    break;
+
+                    case "win":
+                        buttonElement.classList.add('win');
+                        buttonElement.addEventListener('click', () => {
+                            document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                        });
+                        break;
+
+                        case "alt":
+                        buttonElement.classList.add('alt');
+                        buttonElement.addEventListener('click', () => {
+                            document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                        });
+                        break;
+
+
+            
+            
+            default:
+            buttonElement.addEventListener('click', () => {
+                if (this.properties.ru == 'false') {
+                    this.properties.capsLock == false ? this.properties.value += buttonElement.textContent.substring(0,1).toLowerCase() : this.properties.value += buttonElement.textContent.substring(0,1);
+                }
+                else if (this.properties.ru == 'true') {
+                    this.properties.capsLock == false ? this.properties.value += buttonElement.textContent.substring(1,2).toLowerCase() : this.properties.value += buttonElement.textContent.substring(1,2);
+                }
+                
+                document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+
+            })
+
+
+        }
+
+
+            fragment.appendChild(buttonElement);
+
+            if (lineBreak) {
```

Comment:
Was this solution suggested at the lecture? Because I saw it several times.

Reply:
this was in the youtube video, that was pinned in discord chat to get an idea, from what part should i start  begin doing task.